requirements/prod.txt

```diff
@@ -71,6 +71,7 @@ polib==1.0.3
 protobuf==2.5.0
 pyasn1==0.1.7
 PyBrowserID==0.6
+pydenticon==0.2
```

Comment:
is this in our repo? I forget what we use now. I assume it's no longer https://pyrepo.addons.mozilla.org/ because that cert has been revoked :)